tests/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ target_link_libraries(testAll PRIVATE
     libsession::config
     libsodium::sodium-internal
     Catch2::Catch2WithMain
+    oxen::logging
 )
```

Comment:
not about that specific file, but I am not too sure where it would be added.
Do we need a test that validates that a dump not multipart is loaded correctly, and saved as multipart?

Reply:
Dumps themselves will are not multipart-aware; the multipart encoding happens when a config needs to be pushed that is too big to fit in one message, and then gets decoded into a single message again when the parts are reassembled just before the data enters the config object.  The only change to dumps is the (new) temporary storage of incomplete partial multiparts that get stored so that we don't have to received all parts in the same call.  For fully reassembled data, the dumped data should look "as if" libsession-util had no size limit and just received and processed a large config data.